code/modules/clothing/modular/storage.dm

```diff
@@ -0,0 +1,159 @@
+/**
+	Modular armor storage storage attachments
+
+	These are storage attachments that equip into storage slots on modular armor
+*/
+
+
+/** Storage modules */
+/obj/item/armor_module/storage
+	icon = 'icons/mob/modular/modular_armor.dmi'
+	icon_state = "mod_is_bag"
+	var/storage_type = /obj/item/storage/internal/modular
+
+
+/obj/item/storage/internal/modular
+	max_storage_space = 2
+	storage_slots = 2
+	max_w_class = WEIGHT_CLASS_TINY
+	bypass_w_limit = list(
+		/obj/item/clothing/glasses,
+		/obj/item/reagent_containers/food/drinks/flask
+	)
+	cant_hold = list(
+		/obj/item/stack/
```

Comment:
```suggestion
		/obj/item/stack
```
`/obj/item/stack/` is a child of `/obj/item/stack`

Reply:
This is mostly a copy paste from helmets as i haven't gone perfectly through storage other than medical and ammo storage.

That said all the other `cant_hold` have it like this.

Reply:
Looks like a widespread issue, but still an issue.